src/Microsoft.AspNetCore.SignalR.Core/HubEndPoint.cs

```diff
@@ -482,6 +517,13 @@ private bool IsStreamed(HubConnectionContext connection, ObjectMethodExecutor me
                 enumerator = null;
                 return false;
             }
+
+            CancellationToken CreateCancellation()
+            {
```

Comment:
Why is this a local function?

Reply:
Didn't like having the exact same code in both of the above if statements